app/components/Table/index.js

```diff
@@ -229,155 +226,131 @@ export default class Table extends Component<Props, State> {
+                        ? styles.iconActive
+                        : styles.icon
+                    )}
+                  />
+                }
+                contentClassName={styles.checkbox}
                 rootClose
               >
-                <div className={styles.checkbox}>
-                  {filter.map(({ label, value }) => (
-                    <div
-                      key={label}
-                      onClick={() =>
-                        this.onFilterInput(
-                          this.state.filters[dataIndex] === value
-                            ? undefined
-                            : value,
-                          dataIndex
-                        )
-                      }
-                    >
-                      <p key={label}>
-                        <CheckBox
-                          label={label}
-                          value={value === this.state.filters[dataIndex]}
-                        />
-                      </p>
-                    </div>
-                  ))}
-                  <Button
-                    flat
+                {filter.map(({ label, value }) => (
+                  <div
+                    key={label}
                     onClick={() =>
-                      this.setState(
-                        (state) => ({
-                          filters: {
-                            ...state.filters,
-                            [dataIndex]: undefined,
-                          },
-                        }),
-                        () => {
-                          this.toggleFilter(dataIndex);
-                          this.onChange();
-                        }
+                      this.onFilterInput(
+                        this.state.filters[dataIndex] === value
+                          ? undefined
+                          : value,
+                        dataIndex
                       )
                     }
                   >
-                    Nullstill
-                  </Button>
-                </div>
-              </Overlay>
+                    <p key={label}>
+                      <CheckBox
+                        label={label}
+                        value={value === this.state.filters[dataIndex]}
+                      />
+                    </p>
+                  </div>
+                ))}
+                <Button
+                  flat
+                  onClick={() =>
+                    this.setState(
+                      (state) => ({
+                        filters: {
+                          ...state.filters,
+                          [dataIndex]: undefined,
+                        },
+                      }),
+                      () => {
+                        this.toggleFilter(dataIndex);
+                        this.onChange();
+                      }
+                    )
+                  }
+                >
+                  Nullstill
```

Comment:
Should this be a `danger` button? Or does that look ugly and out-of-place?

Reply:
not really related to this pr